python/tests/examples/test_examples.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2025 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pathlib
+import runpy
+
+import pytest
+
+all_examples = list(pathlib.Path(__file__, "../../../examples").resolve().rglob("*.py"))
+
+exclude = [
+    "version.py",
+    "helpers/io.py",
+    # Searx
+    "workflows/web_agent.py",
+    # WatsonX
+    "backend/providers/watsonx.py",
+    # OpenAI
+    "backend/providers/openai_example.py",
```

Comment:
I would like some feedback on this, in the TS implementation it checks for the existence of a ENVVAR and runs these if its found, do we want to do the same? If so I would need some help since I have none of these set up at the moment